docs/_conversion/maintaining.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Maintaining your converted integration
+order: 3
+layout: post-toc
+redirect_from: /conversion/
+---
+
+# Maintaining your integration on the new platform
+
+After you’ve completed the process, you’re ready to continue development on the new platform. If you’re coming from the legacy Web Builder tool, the new UI will be familiar, but with some important differences in how you manage your work.
+
+Locating the current integration
+
+On your “My Integrations” page, make sure you’re working on the copy of your integration in the “Integrations” section - not the “Integrations Built with Legacy Web Builder” section.
+
+![](https://cdn.zappy.app/50e1b6f3dfe56965ffce80cc7ad879cd.png)
+
+## Versions
+
+The most immediate and important difference you’ll notice is that the new platform introduces the concept of [versions](https://platform.zapier.com/docs/versions).  
+
+In the legacy Web Builder, you’d make a clone of your project to make changes to your integration. In the new platform the workflow is similar, but you can manage multiple version of your app for a lot more control and flexibility.  
+
+The top level of your integration is a thin container that houses the title, logo, description of your app.  All partnership metrics and analytics will reference this container.
+
+Inside this container are versions.  Versions hold all the actual workings of your integration - triggers, actions, authentication configuration, etc.  You are free to make big changes from one version to another.  
+
+## New management features: promotion, migration, deprecation
+
+_To make a change to your integration._
+
+- Create a new version by cloning an existing version in the versions section found in the left hand navigation bar.
+- Make your changes to the new version.  
+- If you have invited testers they can create Zaps with this new version before anyone else has access to it.
+- When you’re ready for the world to use your new version, promote it.  Promoting a version makes it the version new users will see when they make Zaps. 
+- If you didn’t make breaking changes, migrate your users to the new version.  Unlike Web Builder, this doesn’t happen automatically. You can control when it happens.  A recommended practice is to migrate percentages of your users and watch for problems in your logs before migrating every one.  If you made breaking changes, you can simply leave users using the older version, or deprecate the old version to force users to reconfigure their Zaps to accommodate your changes.
```

Comment:
Might talk about how they only need to migrate a percentage and check if they're not confident about the compatibility of the change? Or something. I think for a lot of little things, `promote & migrate 100` is perfectly fine